app/_locales/en/messages.json

```diff
@@ -1542,7 +1542,7 @@
     "message": "This account has already been connected to MetaMask"
   },
   "selectAnAccountHelp": {
-    "message": "Select the account to view in MetaMask"
+    "message": "Select an account to view in MetaMask. Don't see your account? $1"
```

Comment:
The addition of a substitution here will break all of our `selectAnAccountHelp` translations. We should remove them all.

Reply:
I'll separate that sentence so we don't lose all our other translations; that's a valuable localized sentence.

Reply:
Huh, interesting solution! That works